### PR TITLE
Prepares release notes 140 m1

### LINKED ISSUE #77: Add 1.4.x preview documentation to the web site
Now that 1.4.0-M1 is out, we should add documentation from the `master` branch to the web site.

The documentation generator has a (currently empty) [`previewVersions`](https://github.com/lagom/lagom.github.io/blob/5ce00dfa468c1daf22fb4dc620fdc58fcf766202/src/main/scala/com/lightbend/lagom/docs/DocumentationGenerator.scala#L35-L36) list for this purpose.

### PR DESCRIPTION
**Pull request by ignasi35**
Adds blog announcement and changelog

Fixes #77

### CODE REVIEW

File: src/blog/lagom-1-4-0-M1.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Lagom 1.4.0-M1 is released!
+date: 2017-06-29
+author_github: ignasi35
+tags: news
+summary: >
+    Lagom 1.4.0-M1 has arrived!
+---
+
+We are thrilled to announce that Lagom is moving fast into upgrading it's underlying stack (Akka, Play and Scala) to corresponding latest versions. This is the biggest change you may expect in the upcoming version 1.4.0 of Lagom. There are still [few issues open](https://github.com/lagom/lagom/milestone/17) and some of these pensind issue could potentially be incompatible with the effort we've done in Lagom 1.4.0-M1 already.
```
Comment (TimMoore) [inline src/blog/lagom-1-4-0-M1.md]: Typo: pensind --> pending

File: src/blog/lagom-1-4-0-M1.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+This is a preview release not ready for production so feel free to use it on a safe environment or a staging environment. We know we want to make some more improvements like using the new [Akka Coordinated shutdown](https://github.com/lagom/lagom/issues/742) or [use Akka's DData instead of persistent Shard Coordinator](https://github.com/lagom/lagom/issues/816). We still have to complete the upgrade to support Scala 2.12 but it's on our radar for the 1.4.0 release.
+
+This release includes several improvements with an important number of community contributions by [erip](https://github.com/erip), [odwrotnie](https://github.com/odwrotnie), [yg-apaza](https://github.com/yg-apaza), [benmccann](https://github.com/benmccann) and  [himani1](https://github.com/himani1) in bugfixing, documentation and even adding features. Thanks all, you rock!
+
```
Comment (TimMoore) [inline src/blog/lagom-1-4-0-M1.md]: I think it's worth highlighting the Slick read-side support specifically. This is a notable feature for Scala users with RDBMS, and if they haven't already noticed that it's in 1.4, noting it here will help them find it.

File: src/blog/lagom-1-4-0-M1.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+We're close to releasing a ConductR version that supports the 1.4.0 family of Lagom so if that's your preferred deployment strategy you will need to wait a few days until we've baked that release.
+
+
+
+## Updating
+
+To update an sbt project, change the version of `lagom-sbt-plugin` in your `project/plugins.sbt` file.
+
+To update a Maven project, change the `lagom.version` property in your top-level `pom.xml`.
```
Comment (TimMoore) [inline src/blog/lagom-1-4-0-M1.md]: We should point to the migration guide as well.

File: src/blog/lagom-1-4-0-M1.md (hunk 2 above)
Comment (TimMoore) [inline src/blog/lagom-1-4-0-M1.md]: The custom service capability in lagom/lagom#763 is also notable.

File: src/blog/lagom-1-4-0-M1.md (hunk 1 above)
Comment (ignasi35) [inline src/blog/lagom-1-4-0-M1.md]: Silly me, had the grammar and spelling check disabled. :-(

File: src/blog/lagom-1-4-0-M1.md (hunk 3 above)
Comment (ignasi35) [inline src/blog/lagom-1-4-0-M1.md]: 👍

**Pull request merged**